TrkHitReco/src/FlagBkgHits_module.cc

```diff
@@ -230,16 +231,26 @@ namespace mu2e
   {
     for (auto& cluster : bkgccol) {
       clusterer_->classifyCluster(cluster,chcol);
-
       StrawHitFlag flag(StrawHitFlag::bkgclust);
       if (cluster.getKerasQ()> kerasQ_) {
         flag.merge(StrawHitFlag(StrawHitFlag::bkg));
         cluster._flag.merge(BkgClusterFlag::bkg);
       }
 
-      for (const auto& chit : cluster.hits()) chfcol[chit].merge(flag);
-
+      for (const auto& chit : cluster.hits())
+        chfcol[chit].merge(flag);
     }
+    /*for (auto& cluster : bkgccol) {
```

Comment:
If this is a useful debug printout I suggest to uncomment it but put it behind a if(_debug>X) condition

Reply:
No, I added it at the time to make sure that the new function "addHitPosition" is functioning correctly. 

But now, we can use the EventDisplay to visually discern which hits belong to which BkgCluster, so we can use that as the debugging tool.